pkg/store/copr/coprocessor.go

```diff
@@ -219,6 +220,15 @@ func (c *CopClient) BuildCopIterator(ctx context.Context, req *kv.Request, vars
 		it.concurrency = 1
 	}
 
+	// if the request is triggered cool down by the runaway checker, we need to adjust the concurrency, let the sql run slowly.\
+	if len(req.ResourceGroupName) > 0 && req.ResourceGroupName != resourcegroup.DefaultResourceGroupName && req.RunawayChecker != nil {
```

Comment:
```suggestion
	if req.RunawayChecker != nil {
```
Why need to check ResourceGroupName?